### ISSUE TITLE
Workspace.eachEditor is deprecated.

### ISSUE THREAD

**Issue by kaisellgren** [Nicolab/atom-local-history#23]
Use Workspace::observeTextEditors instead
```
Workspace.eachEditor (C:\Users\kai\AppData\Local\atom\app-0.190.0\resources\app\src\workspace.js:942:7)
Object.activate (C:\Users\kai\.atom\packages\local-history\lib\local-history.js:34:20)
```

**Comment by Nicolab** [comment]
Ok I will change, thank you

**Comment by Nicolab** [comment]
Released to local-history@v3.0.0 (compatible Atom v0.192 and higher).

Thanks :)
